build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.0.1'
```

Comment:
This could cause issues with fdroid. At least I saw the MB app fail to build there for some Gradle version issue. Do you know more?

Reply:
No. We basically had some lint errors. I have fixed this in the latest commit.